src/axolotl/core/trainer_builder.py

```diff
@@ -1836,6 +1836,7 @@ def build(self, total_num_steps):
         if self.cfg.chat_template:
             training_arguments_kwargs["chat_template"] = get_chat_template(
                 self.cfg.chat_template,
+                jinja_template=self.cfg.chat_template_jinja,
                 tokenizer=self.tokenizer,
             )
```

Comment:
Thanks for the catch, we might have missed in my PR that worked on this, it might make sense to use `get_chat_template_from_config` instead

```suggestion
            training_arguments_kwargs["chat_template"] = get_chat_template_from_config(
                cfg=self.cfg.chat_template,
                tokenizer=self.tokenizer,
            )
```

Reply:
Thanks.. Used your suggestion.